app/uk/gov/hmrc/brm/utils/BRMFormat.scala

```diff
@@ -30,9 +31,17 @@ trait BRMFormat {
 
   private val invalidNameCharsRegEx =  "[;/\\\\(){}&|]|(<!)|(-->)|(\\n)|(\")".r
   private val validBirthReferenceNumberRegEx = """^[a-zA-Z0-9_-]+$"""
```

Comment:
@chris-ianson I'm struggling to understand, I'm referring to ` private val validBirthReferenceNumberRegEx = """^[a-zA-Z0-9_-]+$"""`